src/renderer/src/routes/modals/ConfirmPasswordBackup/index.tsx

```diff
@@ -0,0 +1,147 @@
+            blockchain: acc.blockchain,
+            idWallet: acc.accountType,
+            name: acc.name,
+            order: acc.order,
+            key: acc.encryptedKey ? await window.api.decryptBasedEncryptedSecret(acc.encryptedKey) : undefined,
+          }
+          accountsBackup.push(accBackup)
+        })
+
+      const walletBackup: TWalletBackupFormat = {
+        id: wallet.id,
+        name: wallet.name,
+        walletType: wallet.walletType,
+        mnemonic: wallet.encryptedMnemonic
+          ? await window.api.decryptBasedEncryptedSecret(wallet.encryptedMnemonic)
+          : '',
+        accounts: accountsBackup,
+      }
+      backupFile.wallets.push(walletBackup)
+    })
+
+    const content = await window.api.encryptBasedSecret(JSON.stringify(backupFile), encryptedPassword ?? '')
+
+    window.api.saveFile(`${selectedFilePath}/NEON3-Backup-${DateHelper.todayDateHour()}.txt`, content)
+    modalNavigate('backup-successfull', {
+      state: {
+        iconHeader: <MdOutlineSave className="text-neon" />,
+        textHeader: t('title'),
+        description: t('modalDescription'),
+        showDownloadButton: true,
+      },
+    })
+  }
+
+  return (
+    <EndModalLayout
+      heading={t('title')}
+      headingIcon={<MdOutlineSave className="text-neon" />}
+      contentClassName="flex flex-col"
+    >
+      <p className="text-xs mb-5">{t('description')}</p>
+      <p className="text-gray-300 uppercase font-bold mb-3.5">{t('subtitle')}</p>
+
+      <form className="flex flex-col justify-between flex-grow" onSubmit={handleAct(handleSubmit)}>
+        <div>
+          <Input
+            placeholder={t('inputPlaceholder')}
+            error={!!actionState.errors.password}
+            value={actionData.password}
+            onChange={setDataFromEventWrapper('password')}
+            compacted
+            type="password"
+          />
+
+          <div className="mt-5">
+            {actionState.errors.password && (
+              <div className="bg-magenta-700 rounded flex items-center pl-5 py-2.5 gap-5">
+                <TbAlertTriangle className="text-magenta h-6 w-6" />
+                <span className="text-xs">{actionState.errors.password}</span>
+              </div>
+            )}
+          </div>
+        </div>
+
+        <div className="flex flex-col w-full items-center">
+          <div className={'flex bg-gray-300/15 rounded-md overflow-hidden items-center'}>
+            <div className="flex py-3 px-4 h-full items-center justify-center bg-gray-300/30 relative">
+              <TbAlertSmall className="text-orange w-6 h-6" />
+              <div className="absolute w-4 h-4 border-2 border-orange rotate-45 rounded-sm"></div>
+            </div>
+
+            <p className="px-5 text-xs text-white py-4.5">
+              <span>
+                {t('warning')}
+                <span className="text-orange pl-0.75">{t('warningHighlighted')}</span>
+              </span>
+            </p>
+          </div>
```

Comment:
Why not use Banner component here ? If this variation does not exist create a new one